airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/util/Jsons.kt

```diff
@@ -25,6 +25,7 @@ object Jsons : ObjectMapper() {
         registerModule(AfterburnerModule())
         setSerializationInclusion(JsonInclude.Include.NON_NULL)
         configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+        configure(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS, true)
```

Comment:
Curious: can platform and destination both handle big decimal in json?

Reply:
They should. They're already getting BigDecimals. What this setting does is, when there is no type information, serialized numbers with decimal points in them are turned into BigDecimals instead of doubles.